internal/background/initialize.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // Maximum batch size for each batch send, also an incoming buffered channel size to prevent
 // incoming requests to overload the sender.
-const availabilityStatusBatchSize = 1024
+const availabilityStatusBatchSize = 16
```

Comment:
This was way optimistic. Real data from stage are about dozen of status messages captured per tick. It does not need to be that big.